editor.planx.uk/src/@planx/components/DrawBoundary/Public/index.tsx

```diff
@@ -36,6 +39,10 @@ export default function Component(props: Props) {
   const [slots, setSlots] = useState<FileUploadSlot[]>(previousFile ?? []);
   const [area, setArea] = useState<number | undefined>(previousArea);
   const environment = useStore((state) => state.previewEnvironment);
+  const addressPoint = point([
+    Number(passport?.data?._address?.longitude),
+    Number(passport?.data?._address?.latitude),
+  ]);
```

Comment:
todo: transform longitude, latitude to decimal degrees before making point because not currently working as `clipGeojsonData` yet https://turfjs.org/docs/#point